web/src/typeahead_helper.js

```diff
@@ -398,8 +386,18 @@ export function sort_recipients({
             items = [...items, ...getter()];
         }
     }
+    // Push exact matches to top.
+    const exact_matches = [];
+    const rest = [];
+    for (const item of items) {
+        if (item.full_name?.toLowerCase() === query) {
+            exact_matches.push(item);
+        } else {
+            rest.push(item);
+        }
+    }
```

Comment:
I would expect this block of code to be a noop; aren't exact full name matches already at the top by being the first item in `best_users`?

Reply:
The tests pass if I revert this hunk too.

Reply:
Oops, I ran the wrong test. OK, I see why we need this -- adding a comment to document it for posterity.